packages/manager/modules/vps/src/dashboard/vps-dashboard.routing.js

```diff
@@ -98,6 +103,84 @@ export default /* @ngInject */ ($stateProvider) => {
         $state.go(`vps.detail.dashboard.configuration.upgrade`, {
           upgradeType,
         }),
+
+      vpsMigration: /* @ngInject */ ($http, serviceName) =>
+        $http.get(`/vps/${serviceName}/migration2014`),
+
+      isMigrationRequired: /* @ngInject */ (vpsMigration) =>
+        get(vpsMigration, 'data.status') === MIGRATION_STATUS.TO_PLAN,
+
+      migrationDate: /* @ ngInject */ (vpsMigration) =>
+        moment(get(vpsMigration, 'data.notAfter')).format('DD/MM/YYYY'),
+
+      goToVpsMigration: /* @ngInject */ (
+        $state,
+        vpsMigration,
+        catalog,
+        stateVps,
+        VpsMigrationService,
+      ) => () => {
+        const migrationPlan = VpsMigrationService.constructor.getMigrationPlan(
+          catalog,
+          vpsMigration.data.model,
+        );
+        const server = assign({
+          displayName: stateVps.displayName,
+          name: stateVps.name,
+          description: VpsMigrationService.getVpsModelDescription(
+            stateVps.model,
+          ),
+          migration: VpsMigrationService.constructor.populateOptionsPrice(
+            vpsMigration.data,
+            catalog,
+          ),
+          migrationPlan,
+          migrationDescription: VpsMigrationService.getVpsModelDescription(
+            migrationPlan.description,
+          ),
+        });
+        return $state.go(`vps.detail.dashboard.schedule`, {
+          servers: [server],
+        });
+      },
+    },
+  });
+  $stateProvider.state('vps.detail.dashboard.schedule', {
```

Comment:
Could this be moved to the according module ?

Reply:
This is done